TCMManagement/BusinessLayer/PersonService.cs

```diff
@@ -23,10 +23,11 @@ public Person CreateItem(Person p)
 
         public IEnumerable<Person> GetItems(IEnumerable<KeyValuePair<string, string>> queryParams = null)
         {
-            if(!Utils.IsNullOrEmpty(queryParams)){
+            if(!Utils.IsNullOrEmpty(queryParams)){
+                string roleDescription = queryParams.First().Value;
                 return context.People
                             .Include(p => p.Role)
-                            .Where(e => e.Role.Description == queryParams.First().Value)
```

Comment:
Same reason as I mentioned  in PatientService.cs : Looks like can't put "queryParams.FirstOrDefault().Value" directly to context.Where otherwise will get exception.

Reply:
No worries